src/components/Header/Header.jsx

```diff
@@ -21,29 +21,35 @@ import BurgerMenu from '../BurgerMenu';
 const tablet = 864;
 const mobile = 425;
 
-const navbarPages = [
 // TODO Explorer
 // TODO Matches
-  {
-    name: strings.header_heroes,
-    path: '/heroes',
-  }, {
-    name: strings.header_distributions,
-    path: '/distributions',
-  }, {
-    name: strings.header_ingame,
-    sponsored: true,
-    external: true,
-    path: 'https://dota2.becomethegamer.com/yasp',
-  }];
+const navbarPages = [{
+  name: strings.header_heroes,
+  path: '/heroes',
+}, {
+  name: strings.header_distributions,
+  path: '/distributions',
+}, {
+  name: strings.header_ingame,
+  sponsored: true,
+  external: true,
+  path: '//dota2.becomethegamer.com/yasp',
+}];
+
+const navbarBug = {
+  name: strings.app_report_bug,
+  external: true,
+  path: '//github.com/odota/ui/issues',
+};
 
 const LogoGroup = ({ width }) => {
-  if (width <= mobile) {
-    navbarPages.push({
-      name: strings.app_report_bug,
-      external: true,
-      path: 'https://github.com/odota/ui/issues',
-    });
+  if (width < mobile && navbarPages.map(page => page.path === '//github.com/odota/ui/issues').filter(Boolean).length < 1) {
```

Comment:
How about:

* Save the length of the array
* Push if the navBarPages.length === savedLength